cornice/validators/_colander.py

```diff
@@ -32,12 +32,11 @@ def body_validator(request, schema=None, deserializer=None, **kwargs):
     class RequestSchema(colander.MappingSchema):
         body = _ensure_instantiated(schema)
 
-        def deserialize(self, cstruct=colander.null):
-            appstruct = super(RequestSchema, self).deserialize(cstruct)
-            return appstruct['body']
-
     validator(request, RequestSchema(), deserializer, **kwargs)
 
+    if 'body' in request.validated:
```

Comment:
This change is needed so we don't break the `request.validated.update(deserialized)` call on the base validator.
https://github.com/gabisurita/cornice/blob/14e783a9189c3557b388cc5183ab4490b9bea491/cornice/validators/_colander.py#L80